pom.xml

```diff
@@ -141,6 +141,17 @@
     </properties>
 
     <profiles>
+        <profile>
+            <id>HDP-3.1.0.0</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <hdp_version>3.1.0.0</hdp_version>
+                <global_storm_version>1.2.1</global_storm_version>
```

Comment:
Can we separate Storm and Kafka? These are not closely related the way ES and Kibana are.

Reply:
Sure.  I don't believe any changes were needed for Storm.